src/main.js

```diff
@@ -26,6 +34,7 @@ rdmCoverBtn.addEventListener('click', showRdmCover);
 function getRandomIndex(array) {
   return Math.floor(Math.random() * array.length);
 };
+
 function showRdmCover() {
```

Comment:
We might want to refactor this function into two functions, one for generating a random cover and one for rendering the cover.